contracts/metadata/metadata.cairo

```diff
@@ -0,0 +1,647 @@
+        const wonder_49 = 27352061535142984367278971188825042546277; // Pagoda Of Fortune
+        const wonder_50 = 438270318759684835528781643805323116; // The Oracle Pool
+    }
+}
+
+namespace Uri {
+    func build{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
+        realm_id: Uint256, realm_data: RealmData, realm_type: felt
+    ) -> (encoded_len: felt, encoded: felt*) {
+        alloc_locals;
+
+        // pre-defined for reusability
+        let left_bracket = Utils.Symbols.LeftBracket;
+        let right_bracket = Utils.Symbols.RightBracket;
+        let inverted_commas = Utils.Symbols.InvertedCommas;
+        let comma = Utils.Symbols.Comma;
+
+        let data_format = 37556871985679581542840396273993309325169359621942828; // data:application/json,
+
+        let description_key = 697556140257669248036922561798714; // "description":
+        let name_key = 9691513934389818; // "name":
+        let image_key = 2479633334958105146; // "image":
+        let attributes_key = 2723918356737767845177276441146; // "attributes":
+
+        let left_square_bracket = 91;
+        let right_square_bracket = 93;
+        // get value of description
+        let description_value = 2482157813739909922; // "realms"
+
+        // get name
+        let name_value = 2480746070689997090; // "mahala"
+
+        let image_url_1 = 7526768903561293615;  // https://
+        let image_url_2 = 138296299598040277212951397;  // realms-asse
+        let image_url_3 = 2361885128291351757209958612169773;  // ts.s3.eu-west-
+        let image_url_4 = 265747849610518128817807013032718189;  // 3.amazonaws.com
+        let image_url_5 = 875240087534798271279;  // /renders/
+
+        let trait_key = 639351341392214529084052888304630306;  // {"trait_type":"
+        let value_key = 635719516926804900386;  // "value":"
+
+        let regions_key = 1519939938891930477100;  // Regions",
+
+        let cities_key = 4857541669118222892;  // Cities",
+
+        let harbors_key = 1335187356306912780844;  // Harbors",
+
+        let rivers_key = 5938407761748632108;  // Rivers",
+
+        let resource_key = 389105490742926503125548;  // Resource",
+
+        let (values: felt*) = alloc();
+        assert values[0] = data_format;
+        assert values[1] = left_bracket;  // start
+        // description key
+        assert values[2] = description_key;
+        assert values[3] = description_value;
+        assert values[4] = comma;
+        // name value
+        assert values[5] = name_key;
+        assert values[6] = name_value;
+        assert values[7] = comma;
+        // image value
+        assert values[8] = image_key;
+        assert values[9] = inverted_commas;
+        assert values[10] = image_url_1;
+        assert values[11] = image_url_2;
+        assert values[12] = image_url_3;
+        assert values[13] = image_url_4;
+        if (realm_type == 1) {
```

Comment:
Can you make `const`ants out of the realm type 1 and 2? I don't actually know what they are 😄